packages/web/src/helpers/createEndpoint.js

```diff
@@ -0,0 +1,61 @@
+// Module imports
+import cors from 'micro-cors'
```

Comment:
I don't really know what this is doing, and I've run out of time to actually review

Reply:
Haha, this is part of the [Next.js API routes](https://nextjs.org/docs/api-routes/introduction) stuff. We use the `micro-cors` library to add support for CORS requests. By default, `micro-cors` will allow requests to the API from pretty much anywhere. Before we go to prod, we'll definitely want to lock that down to only trusted domains.